### PR TITLE
DOCS: update gpconfig. Updated, simplified quoting syntax for GUC values

### PR DESCRIPTION
**Pull request by mkiyama**
Updated description and examples to use new syntax.

port of reviewed 4.3.x content.

### CODE REVIEW

**Comment by mkiyama** [comment]
@Chibin If you haven't already, could you take a quick look at the updated gpconfig examples.
I just changed them to have single quotes around the GUC values.
Thanks

File: gpdb-doc/dita/utility_guide/admin_utilities/gpconfig.xml [hunk 1]
```diff
@@ -170,13 +176,18 @@
...
-        When you specify the values for the parameters, enclose the values with double quotes
-          (<codeph>"</codeph>).<codeblock>$ gpconfig -c gp_email_from -v "'gpdb-server@example.com'"
-$ gpconfig -c gp_email_to -v "'gpdb-admin@example.com'"</codeblock></p>
+        <codeblock>$ gpconfig -c gp_email_from -v 'gpdb-server@example.com'
+$ gpconfig -c gp_email_to -v 'gpdb-admin@example.com'</codeblock></p>
       <p>In the <codeph>postgresql.conf</codeph> file, the parameters are set correctly, with single
         quotes around the
         values:<codeblock>gp_email_from='gpdb-server@example.com'
 gp_email_to='gpdb-admin@example.com'</codeblock></p>
+      <p>These example shows the syntax required due to bash shell string processing.</p>
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpconfig.xml]: example -> examples

File: gpdb-doc/dita/utility_guide/admin_utilities/gpconfig.xml [hunk 2]
```diff
@@ -170,13 +176,18 @@
...
+      <codeblock>gpconfig -c search_path -v '"\$user",public'
+gpconfig -c dynamic_library_path -v '\$libdir'</codeblock>
+      <p>The configuration paramters are added to the <codeph>postgresql.conf</codeph> file.</p>
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpconfig.xml]: Typo:  paramters

**Comment by mkiyama** [comment]
@dyozie fixed the typos.

**Pull request merged**